en/docs/compliance/onc-certification.md

```
@@ -0,0 +1,142 @@
+| Requirement | Implementation |
+|-------------|---------------|
+| FHIR R4 APIs | US Core IG profiles |
+| SMART App Launch | OAuth 2.0 authorization |
+| Bulk Data Access | Backend Services IG |
+| Patient-level access | SMART scopes (patient/*.read) |
+| System-level access | SMART Backend Services |
+| Multi-patient queries | Group/$export |
+
+### Required SMART Scopes
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)